account-kit/smart-contracts/src/ma-v2/permissionBuilderErrors.ts

```diff
@@ -151,3 +151,16 @@ export class UnsupportedPermissionTypeError extends BaseError {
     super(`Unsupported permission type`);
   }
 }
+
+export class SelectorNotAllowed extends BaseError {
+  override name = "SelectorNotAllowed";
+
+  /**
+   * Constructor for initializing an error message indicating that the selector being added is not allowed.
+   *
+   * @param {string} selector The selector that is being added.
+   */
+  constructor(selector: string) {
+    super(`Cannot add ${selector} on the account`);
```

Comment:
It would probably be more helpful for debugging to include the human-readable function name in this exception